### PR TITLE
WIP: wicked IPv6: add second test case , fix issues in first one

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

**Comment by schlad** [comment]
hmm somehow it looks like the PR/git commit messages do not match what I see in the code.

File: lib/wickedbase.pm [hunk 1]
```diff
@@ -542,19 +542,34 @@ sub check_fail_over {
...
+  sync_start_of($service, $mutex, [,$timeout])
+
+Start C<$service> within defined $timeout ( default is 60).
+After succesfully service start will create mutex with C<$mutex>
+which can be used by parallel test to catch this event
+
+=cut
+sub sync_start_of {
+    my ($self, $service, $mutex, $timeout) = @_;
+    my $timeout //= 60;
```
Comment (cfconrad) [inline lib/wickedbase.pm]: I Have the feeling you `$timeout` looks like seconds. But is a `try` count.

File: lib/wickedbase.pm [hunk 2]
```diff
@@ -542,19 +542,34 @@ sub check_fail_over {
...
+            record_info("$service is active", "$service is active");
+            die("Create mutex failed") unless mutex_create($mutex);
+            systemctl("status $service.service");
+            return;
+        }
+        else {
+            record_info('WAIT 5s', "$service still starting");
+            $timeout -= 1;
+            sleep 5;
         }
```
Comment (cfconrad) [inline lib/wickedbase.pm]: What about something like:
```
assert_script_run("while ! systemctl is-active $service.service; do sleep 5; done", timeout=>$timeout);
die("Create mutex failed") unless mutex_create($mutex);
systemctl("status $service.service");
```

Comment (asmorodskyi) [reply lib/wickedbase.pm]: make sense

**Pull request closed without merging**